source/common/access_log/access_log_formatter.cc

```diff
@@ -136,10 +146,13 @@ std::vector<FormatterPtr> AccessLogFormatParser::parse(const std::string& format
 
         formatters.emplace_back(
             FormatterPtr(new ResponseHeaderFormatter(main_header, alternative_header, max_length)));
+      } else if (token.find("START_TIME(") == 0) {
+        const size_t start = 11;
+        const std::string args = token.substr(start, command_end_position - (pos + start + 1));
+        formatters.emplace_back(FormatterPtr(new RequestInfoFormatter("START_TIME", args)));
```

Comment:
To keep with the pattern already established here, because more arguments are needed, I think you should create a new Formatter for this: StartTimeFormatter.